Appraisals

```diff
@@ -23,16 +24,15 @@ end
 
 appraise "ar4" do
   gem "activerecord", "~> 4.2"
+  gem "sinatra", "~> 1.4.6"
 end
 
 appraise "ar5" do
   gem "activerecord", "~> 5.0.0"
   gem "rspec-rails", "~> 3.5.1"
   gem 'rails-controller-testing'
 
-  # Sinatra stable conflicts with AR5's rack dependency. Sinatra master requires
-  # rack-protection master. Specify exact `ref` so it doesn't break in the future.
-  # Hopefully there'll be a sinatra 2.0 release soon.
-  gem 'sinatra', github: 'sinatra/sinatra', ref: "a7483f48b0a18ba792e642a"
-  gem "rack-protection", github: "sinatra/rack-protection", ref: "7e723a74763bb83989d12"
+  # The AR5 version of PaperTrail is not compatible with sinatra 2 yet.
+  # Contributions welcome.
```

Comment:
Thanks for taking a look. I tried testing against sinatra 2.0.0.beta2 but got something like "url_helpers undefined". If you want to tinker with it, I can make you a branch that reproduces the error.